### DISCUSSION TITLE
Slash commands not working on replit

### DISCUSSION THREAD

**Discussion by testbot-github** [SudhanPlayz/Discord-MusicBot#634] [Q&A] [answered]
**Please describe the problem you are having in as much detail as possible:**
I imported this repo into replit and changed to v5 branch, then after I filled the config I ran it.
Then after some time it told loaded slash commands, but when I checked the server there were no slash commands for the bot even though I invited the bot with slash commands perms.

**Further details:**

- discord.js version: 13.2.0
- Node.js version: 16.13.1
- Operating system: Replit

**Relevant client options:**

- partials: none
- gateway intents: presence and server members intent
- invite perms: applications.commands and bot
- other: none

**Comment by LewdHuTao** [comment]
go to shell/console and type `node deploy/global` for all guilds or type `node deploy/guild` for one guild

**Reply by testbot-github** [reply]
Looks like that will not work cuz in replit u need to use `npx node` for discord.js@v13 so the answer will be:
Go to shell/console and type `npx node deploy/global` for all guilds or type `npx node deploy/guild` for one guild

**Comment by testbot-github** [comment, marked as answer]
Looks like that will not work cuz in replit u need to use `npx node` for discord.js@v13 so the answer will be:
Go to shell/console and type `npx node deploy/global` for all guilds or type `npx node deploy/guild` for one guild

**Comment by DarrenOfficial** [comment]
This has been covered by the v5 docs, please refrain for making spam discussion like this in the future.
